rules/android/android_library.bzl

```diff
@@ -77,12 +85,12 @@ def android_library(
         assets = attrs.get("assets", default = None),
         assets_dir = attrs.get("assets_dir", default = None),
         visibility = attrs.get("visibility", default = None),
-        tags = attrs.get("tags", default = None),
+        tags = attrs.get("tags", default = []) + ["lint_enabled"],
         deps = android_library_deps,
         plugins = attrs.get("plugins", default = None),
     )
 
-    lint(
+    lint_test(
```

Comment:
would passing a `test = True` here makes it more explicit that we should run this via bazel test?

Reply:
Actually I don't think we need it here since the rule itself is made a test rule 

https://github.com/arunkumar9t2/grab-bazel-common/blob/30760c05207efe1d4b72fb7c022bfefa57968932/rules/android/lint/lint.bzl#L326

Reply:
ohh yeah missed that one out. i think i got confused with the rule itself